packages/SwingSet/src/lib-nodejs/spawnSubprocessWorker.js

```diff
@@ -23,14 +23,16 @@ function parentLog(first, ...args) {
 // always be Node.
 const stdio = harden(['inherit', 'inherit', 'inherit', 'pipe', 'pipe']);
 
+const NETSTRING_MAX_CHUNK_SIZE = 12_000_000;
```

Comment:
let's see, this limit will be imposed on all workers that use subprocesses (xsnap, node), ok

If the limit is exceeded, it will basically look like the child process had an error (a malformed netstring, or more likely the child closed the pipe because it's in the process of exiting). I don't know that we're handling that as well as we could be, but fixing that is independent of this change. Given the similarity between this error and ones that can be triggered by e.g. OS shutdown, we should remember to treat these as non-consensus errors, and panic the kernel rather than consensus-terminate the vat.